src/core/i18n/translation/locales/en.ts

```diff
@@ -74,7 +86,12 @@ export const translation: ITranslations = {
             inputAddress: 'Search public addresses or domain names',
             amount: 'Amount',
             recipientLabel: 'Recipient',
-            transferOwnAccounts: 'Transfer between my accounts'
+            transferOwnAccounts: 'Transfer between my accounts',
+            advancedSetup: 'Advanced setup',
```

Comment:
this text is duplicated

Reply:
also, i think we should use `Advanced` simple. Same for `Simple`. 
Move them to labels.